docusaurus/sidebar-connectors.js

```diff
@@ -83,14 +83,14 @@ function loadConnectorRegistry() {
 
 function addSupportLevelToConnectors(connectors, registry) {
   return connectors.map((item) => {
-    if (item.type !== "doc" || !item.id) {
+    // Get the ID from either doc-type or category-type items
+    const id = item.type === "doc" ? item.id : item.link && item.link.id;
+
+    if (!id) {
       return item;
     }
 
-    const idParts = item.id.split("/");
-    const connectorName = idParts[idParts.length - 1];
-
-    const connectorInfo = registry.find((c) => c && c.id === connectorName);
+    let connectorInfo = registry.find((record) => record.docUrl.includes(id));
```

Comment:
Since we already had `docUrl`info on `connector_registry_slim.json` 
 I've simplified connector matching to use that property, this ensures we are following the same criteria since we read `docs/... ` folder it made more sense to rely on the `docUrl`